translations/bg/LC_MESSAGES/messages.po

```diff
@@ -1,167 +1,104 @@
-
 #, fuzzy
 msgid "Access Before Assign"
-msgstr ""
-"You tried to use the variable {name} on line {access_line_number}, but "
-"you set it on line {definition_line_number}. Set a variable before using "
-"it."
+msgstr "You tried to use the variable {name} on line {access_line_number}, but you set it on line {definition_line_number}. Set a variable before using it."
 
 #, fuzzy
 msgid "Cyclic Var Definition"
-msgstr ""
-"The name {variable} needs to be set before you can use it on the right-"
-"hand side of the is command"
+msgstr "The name {variable} needs to be set before you can use it on the right-hand side of the is command"
 
 msgid "Has Blanks"
-msgstr ""
-"Кодът ти е непълен. Съдържа празни места, който трябва да замениш с "
-"програмен код."
+msgstr "Кодът ти е непълен. Съдържа празни места, който трябва да замениш с програмен код."
 
 msgid "Incomplete"
-msgstr ""
-"Момент, кодът ти не е пълен! На ред {line_number} е нужно да напишеш "
-"текст след {incomplete_command}."
+msgstr "Момент, кодът ти не е пълен! На ред {line_number} е нужно да напишеш текст след {incomplete_command}."
 
 #, fuzzy
 msgid "Incomplete Repeat"
-msgstr ""
-"It looks like you forgot to use {command} with the repeat command you "
-"used on line {line_number}."
+msgstr "It looks like you forgot to use {command} with the repeat command you used on line {line_number}."
 
 msgid "Invalid"
-msgstr ""
-"{invalid_command} не е валидна команда в ниво {level} на Хеди. Може би "
-"имаш предвид {guessed_command}?"
+msgstr "{invalid_command} не е валидна команда в ниво {level} на Хеди. Може би имаш предвид {guessed_command}?"
 
 #, fuzzy
 msgid "Invalid Argument"
-msgstr ""
-"You cannot use the command {command} with {invalid_argument} . Try "
-"changing {invalid_argument} to {allowed_types}."
+msgstr "You cannot use the command {command} with {invalid_argument} . Try changing {invalid_argument} to {allowed_types}."
 
 #, fuzzy
 msgid "Invalid Argument Type"
-msgstr ""
-"You cannot use {command} with {invalid_argument} because it is "
-"{invalid_type}. Try changing {invalid_argument} to {allowed_types}."
+msgstr "You cannot use {command} with {invalid_argument} because it is {invalid_type}. Try changing {invalid_argument} to {allowed_types}."
 
 #, fuzzy
 msgid "Invalid At Command"
-msgstr ""
-"The {at} command may not be used from level 16 onward. You can use square"
-" brackets to use an element from a list, for example `friends[i]`, "
-"`lucky_numbers[{random}]`."
+msgstr "The {at} command may not be used from level 16 onward. You can use square brackets to use an element from a list, for example `friends[i]`, `lucky_numbers[{random}]`."
 
```

Comment:
It happened again :c Can you tell the exact command you're using, @Annelein?

Reply:
Nevermind, I reproduced the issue! Will try figure this one out. Don't worry about this and continue working on the actual implementation.

Reply:
```
pybabel extract -F babel.cfg -o messages.pot . --no-location --sort-output --omit-header
pybabel update -i messages.pot -d translations -N --no-wrap --omit-header
pybabel compile -f -d translations
```

Reply:
thank you!

Reply:
Yeah, your command is working as we expected, it's weird that main has the lines wrapped and is causing these conflicts